### PR TITLE
image label with tooltip

### LINKED ISSUE #260: UX improvement of image name
- **Feature**
  - [ ] feature request

- **What is the current behavior?**
image

- **What is the expected behavior?**
UX improvement-
- Color code of the image display name can be corrected.
- Responsive design
- tooltip on hover, shows complete name.

### PR DESCRIPTION
**Pull request by ashishnagpal2498**
## Fixes issue fixes #260 

Closes: #260 
- Color code matches with logo 
- Overflow handled with CSS
- on hover - complete name Tooltip
- Responsive case handled 
image

### CODE REVIEW

**Review by rajats98 — CHANGES_REQUESTED** [review]
Awesome work!, just make these small changes
Align the text span vertically in the middle
The color choice is good but needs to be a bit more contrasting for easy readability
decrease vertical padding, (you can also decrease the border-radius to make it squarish)

**Review by rajats98 — APPROVED** [review]

**Comment by ashishnagpal2498** [comment]
Hey, @rajats98 , I think this was a medium difficulty level task. Also the unnecessary code was removed in PR #266 , therefore "hard". Increase the label whatever feels suitable to you. Thanks 😄

**Comment by rajats98** [comment]
Your work is great! I forgot about #266 ! updated to medium. :100: !

File: src/assets/index.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ const analyze_click = () => {
     document.getElementById("leaf_input").value = "";
     document.getElementById("showImage").src = DUMMY_URL;
     uploadButtonSpan.innerHTML = "Upload a file";
-    document.getElementById("img-lab").innerHTML = "";
```
Comment (rajats98) [inline src/assets/index.js:119]: Hey, Can you check and update the behavior of img-label when analyze other crop is clicked from result page!
I think only a few changes are needed here! create a new PR and refer this, you will not get any points for new one :sweat_smile:  !

Comment (ashishnagpal2498) [reply src/assets/index.js:119]: Okay. Yeah sure, didn't consider that case.

**Pull request merged**